### PR TITLE
Restructure repo

### PR DESCRIPTION
**Pull request by folex**
```
.
├── LICENSE
├── README.md
├── aqua
│   ├── dht-scheduled-scripts.aqua
│   ├── dht.aqua
│   ├── package-lock.json
│   ├── package.json
│   ├── pubsub.aqua
│   └── target
│       └── typescript
│           ├── dist
│           ├── package-lock.json
│           ├── package.json
│           ├── src
│           └── tsconfig.json
├── examples
│   ├── aqua
│   │   ├── example.aqua
│   │   ├── example.ts
│   │   ├── package-lock.json
│   │   └── package.json
│   └── typescript
│       ├── dist
│       │   └── index.js
│       ├── example.ts
│       ├── package-lock.json
│       ├── package.json
│       └── tsconfig.json
└── service
    ├── Cargo.lock
    ├── Cargo.toml
    ├── Config.toml
    ├── README.md
    ├── artifacts
    │   └── aqua-dht.wasm
    ├── build.sh
    ├── rust-toolchain
    └── src
        ├── impls.rs
        ├── main.rs
        ├── results.rs
        └── tests.rs
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -4,35 +4,35 @@ orbs:
...
       RUST_BACKTRACE: full
+    working_directory: ./service
     steps:
       - checkout
       - run: |
           sudo bash .github/download_marine.sh
       - restore_cache:
           keys:
-            - aqua-dht04-{{ checksum "Cargo.lock" }}
+            - aqua-dht04-{{ checksum "service/Cargo.lock" }}
```
Comment (justprosh) [inline .circleci/config.yml:20]:
```suggestion
            - aqua-dht05-{{ checksum "service/Cargo.lock" }}
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -4,35 +4,35 @@ orbs:
...
-          rustup target add wasm32-wasi --toolchain nightly-2021-04-24-x86_64-unknown-linux-gnu
       - run: ./build.sh
       - run: |
           cargo test --no-fail-fast --release --all-features -- --test-threads=1
       - save_cache:
           paths:
             - ~/.cargo
             - ~/.rustup
-          key: aqua-dht04-{{ checksum "Cargo.lock" }}
+          key: aqua-dht04-{{ checksum "service/Cargo.lock" }}
```
Comment (justprosh) [inline .circleci/config.yml:32]:
```suggestion
          key: aqua-dht05-{{ checksum "service/Cargo.lock" }}
```

File: aqua/target/typescript/src/dht-scheduled-scripts.ts [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+            })
+        if(config && config.ttl) {
+            r.withTTL(config.ttl)
+        }
+        request = r.build();
+    });
+    await client.initiateFlow(request!);
+    return Promise.race([promise, Promise.resolve()]);
+}
+
```
Comment (justprosh) [inline aqua/target/typescript/src/dht-scheduled-scripts.ts:137]: missing newline

**Review by justprosh — APPROVED** [review]

File: aqua/target/typescript/src/dht-scheduled-scripts.ts (hunk 3 above)
Comment (folex) [reply aqua/target/typescript/src/dht-scheduled-scripts.ts:137]: that's what Aqua compile generates

**Pull request merged**